mlflow/genai/label_schemas/label_schemas.py

```diff
@@ -1,114 +1,155 @@
-from typing import TYPE_CHECKING, Literal, Optional, Union
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Optional, TypeVar, Union
 
 from mlflow.genai.utils.enum_utils import StrEnum
 
 if TYPE_CHECKING:
-    from databricks.agents.review_app.label_schemas import (
-        InputCategorical as _InputCategorical,
-    )
-    from databricks.agents.review_app.label_schemas import (
-        InputCategoricalList as _InputCategoricalList,
-    )
-    from databricks.agents.review_app.label_schemas import (
-        InputNumeric as _InputNumeric,
-    )
-    from databricks.agents.review_app.label_schemas import (
-        InputText as _InputText,
-    )
-    from databricks.agents.review_app.label_schemas import (
-        InputTextList as _InputTextList,
-    )
-    from databricks.agents.review_app.label_schemas import (
-        LabelSchema as _LabelSchema,
-    )
-
-
-class InputCategorical:
+    from typing import TypeAlias
```

Comment:
TypeAlias is only available in python >= 3.10. We don't reach here at runtime, but can we avoid using it just in case? MLflow 3.0's minimum supported version is still 3.9.

Reply:
Absolutely!